pom.xml

```diff
@@ -45,7 +45,9 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <slf4j.version>1.7.36</slf4j.version>
+    <slf4j.version>2.0.13</slf4j.version>
+    <maven.compiler.source>17</maven.compiler.source>
+    <maven.compiler.target>17</maven.compiler.target>
```

Comment:
This is wrong. Look into the parent (up or velocity master), there is a single property for the version.

Reply:
I know, but the enforcer plugin looks at maven.compiler.target for bytecode compatibility

Reply:
You don't understand, I mean this property: https://github.com/apache/velocity-master/blob/e2079f580255539f89b6273c09f0192948460caf/pom/pom.xml#L43